lib/ts/xmlhttprequest.ts

```diff
@@ -240,7 +248,12 @@ export function addInterceptorsToXMLHttpRequest() {
                 } else {
                     // Here we only need to handle fetch related errors, from the refresh endpoint called by the retry
                     // So we should only get network level errors here
-                    redispatchEvent("error", new Event("error"));
+
+                    const ev = new Event("error");
+                    (ev as any).error = err;
+                    self.dispatchEvent(ev);
+
+                    redispatchEvent("error", ev);
```

Comment:
This looks strange, why do we need both calls?

Reply:
We decided to call the `onerror` method directly, instead.